file-formats/cfdf/type/zif_aff_cfdf_v1.intf.abap

```diff
@@ -0,0 +1,592 @@
+      external_namespace   TYPE c LENGTH 255,
+
+      "! <p class="shorttext">External Field Name</p>
+      "! External field name
+      "! $required
+      "! $maxLength 120
+      external_field_name  TYPE c LENGTH 120,
+    END OF ty_gs_soap_usage.
+  TYPES: ty_gt_soap_usage TYPE STANDARD TABLE OF ty_gs_soap_usage
+                          WITH KEY business_context service_interface operation message_direction component_identifier.
+
+  TYPES:
+    "! <p class="shorttext">Code Description</p>
+    "! Code description
+    BEGIN OF ty_gs_codelistvalue_descr,
+      "! <p class="shorttext">Language</p>
+      "! Language
+      "! $required
+      "! $maxLength 2
+      language    TYPE sy-langu,
+
+      "! <p class="shorttext">Code</p>
+      "! Code
+      "! $required
+      "! $maxLength 30
+      code        TYPE ty_code_value,
+
+      "! <p class="shorttext">Description</p>
+      "! Description
+      "! $required
+      "! $maxLength 60
+      description TYPE c LENGTH 60,
+    END OF ty_gs_codelistvalue_descr.
+  TYPES: ty_gt_codelistvalue_descr TYPE STANDARD TABLE OF ty_gs_codelistvalue_descr WITH KEY language code.
+
+  TYPES:
+    "! <p class="shorttext">Code Value</p>
+    "! Code value
+    BEGIN OF ty_gs_codelistvalue,
+      "! <p class="shorttext">Code</p>
+      "! Code
+      "! $required
+      "! $maxLength 30
+      code        TYPE ty_code_value,
+
+      "! <p class="shorttext">Disabled</p>
+      "! Disabled
+      is_disabled TYPE abap_bool,
+    END OF ty_gs_codelistvalue.
+  TYPES: ty_gt_codelistvalue TYPE STANDARD TABLE OF ty_gs_codelistvalue WITH KEY code.
+
+  TYPES:
+    "! <p class="shorttext">Value Help Key Mapping</p>
+    "! Value help key mapping
+    BEGIN OF ty_gs_value_help_key_mapping,
+      "! <p class="shorttext">Component Identifier</p>
+      "! Component identifier
+      "! $maxLength 5
+      component_identifier      TYPE ty_component_identifier,
+
+      "! <p class="shorttext">Value Help Field Name</p>
+      "! Value help field name
+      "! $required
+      "! $maxLength 30
+      value_help_field_name    TYPE ty_field_name,
+
+      "! <p class="shorttext">Dimension View Field Name</p>
+      "! Dimension view field name
+      "! $maxLength 30
+      dimension_view_field_name TYPE ty_field_name,
+    END OF ty_gs_value_help_key_mapping.
+
+  TYPES ty_gt_value_help_key_mapping TYPE STANDARD TABLE OF ty_gs_value_help_key_mapping WITH KEY component_identifier.
+
+
+  TYPES:
+    "! <p class="shorttext">Value Help Binding</p>
+    "! Value help binding
+    BEGIN OF ty_gs_value_help_binding,
+      "! <p class="shorttext">Extended CDS View Name</p>
+      "! Extended CDS view name
+      "! $required
+      "! $maxLength 30
+      cds_view_name         TYPE  ty_cds_view_name,
+
+      "! <p class="shorttext">Extended CDS View Field Name</p>
+      "! Extended CDS view field name
+      "! $required
+      "! $maxLength 30
+      cds_view_field_name   TYPE ty_field_name,
```

Comment:
Just a heads-up: I am not sure how important it is to add `extended` to the field names.

Reply:
Done